docs/howto/auth/github.rst

```diff
@@ -55,13 +55,16 @@ For more information on ``tljh-config``, see :ref:`topic/tljh-config`.
    .. image:: ../../images/notebook/new-terminal-button.png
       :alt: New terminal button.
 
-#. Configure the GitHub OAuthenticator to use your client ID and secret with the following commands::
+#. Configure the GitHub OAuthenticator to use your client ID, client secret and callback URL with the following commands::
 
      sudo tljh-config set auth.GitHubOAuthenticator.client_id '<my-tljh-client-id>'
 
    ::
 
      sudo tljh-config set auth.GitHubOAuthenticator.client_secret '<my-tljh-client-secret>'
+   ::
```

Comment:
```suggestion
   
   ::
```

Reply:
@huhuhang I think this should fix the documentation test failure and we can merge this afterewards